lang/cs/Org.Apache.REEF.IMRU.Examples/MapperCount/MapperCount.cs

```diff
@@ -43,7 +49,7 @@ private MapperCount(IIMRUClient imruClient)
         /// Runs the actual mapper count job
         /// </summary>
         /// <returns>The number of MapFunction instances that are part of the job.</returns>
-        public int Run(int numberofMappers)
+        public int Run(int numberofMappers, string outputFile, IConfiguration fileSystemConfig)
```

Comment:
If client has supplied FileSystemConfiguration when inject REEFClient, in your case, IMRUClient, It will be automatically passed to Driver and Evaluators as we have make it as a provider. You don't need to set it on IMRUJobDefinitionBuilder as part of ResultHandlerConfiguration. You should be able to simply put IFileSystem in any of you injected constructor and use it.